transforms/vactubes/__init__.py

```diff
@@ -233,6 +243,15 @@ async def vactube_transform(ctx: Context) -> None:
         with open(temp_dir + '/prop.qc', 'w') as qc_file:
             qc_file.write(QC_TEMPLATE.format(path=anim_mdl_name.as_posix()))
 
+            # Create an idle sequence, needed for animations not to lag.
+            with open(temp_dir + '/idle.smd', 'wb') as idle_seq_file:
+                mesh.export(idle_seq_file)
+
+            with open(temp_dir + '/idle.smd', 'w') as idle_seq_file:
+                idle_seq_file.write(animations.create_idle_seq())
```

Comment:
The extra file isn't necessary, we can just re-use the reference SMD since it also has a 1-frame animation.